### DISCUSSION TITLE
VcDrawings 绘制 按钮的文字啊其他样式问题   看文档没咋理解  这样写 是不对的吗？

### DISCUSSION THREAD

**Discussion by IKE1001** [zouyaoji/vue-cesium#590] [General]
![image](https://github.com/user-attachments/assets/1fbae6cc-aabb-4370-a53b-a5b86b298303)
![image](https://github.com/user-attachments/assets/4a7f595a-ea09-45ec-9a07-b1ce2e53aa0e)
![image](https://github.com/user-attachments/assets/3119ce85-da62-4e1f-9821-474fd538ea99)

**Comment by zouyaoji** [comment]
你这个是用了插槽的，按钮样式就自己把握咯。

**Reply by IKE1001** [reply]
抱歉 我描述的有误 我的意思是 我自定义的插槽按钮 想要修改 自定义 这个 绘制的 点 线 面  这个 文字

**Reply by zouyaoji** [reply]
你都自己写插槽了 文字也可以自定义呀
